core/isolate.rs

```diff
@@ -85,6 +118,7 @@ pub struct Isolate {
   needs_init: bool,
   shared: SharedQueue,
   pending_ops: FuturesUnordered<OpAsyncFuture>,
+  pending_dyn_imports: FuturesUnordered<DynImport>,
```

Comment:
Yes! This is the right way to do it!

Reply:
I think it would be nice if we could somehow present the incoming imports as a Stream<SpecifierReferrer> and map that into mod_id...